TreeMaker/python/makeTreeFromMiniAOD_cff.py

```diff
@@ -105,6 +106,17 @@ def makeTreeFromMiniAOD(
             isLHE = cms.bool(False)
         )
         VarsDouble.extend(['SusyScan:SusyMotherMass','SusyScan:SusyLSPMass'])
+
+    if pmssm:
+        # pMSSM ID for identifying the pMSSM model point
+        from TreeMaker.Utils.pmssm_cfi import PmssmProducer
+        process.Pmssm = PmssmProducer.clone(
+            shouldScan = cms.bool(fastsim),
```

Comment:
The idea was to keep the pMSSM branches in all MC ntuples (to avoid unnecessary digressions in the ntuple format), but only fill them when the pmssm parameter is activated, i.e.:
```
shouldScan = cms.bool(pmssm)
```
and remove the `if pmssm` above.